Python/bltinmodule.c

```diff
@@ -1821,23 +1821,31 @@ builtin_pow_impl(PyObject *module, PyObject *base, PyObject *exp,
     return PyNumber_Power(base, exp, mod);
 }
 
+/*[clinic input]
+print as builtin_print
+
+    *args: object
+    sep: object(c_default="NULL") = ' '
+    end: object(c_default="NULL") = '\\n'
+    file: object(c_default="NULL") = None
```

Comment:
```suggestion
    file: object = None
```
And remove the special case for NULL below.